### ISSUE TITLE
Pass argument from test case to fixture

### ISSUE THREAD

**Issue by The-Compiler** [hackebrot/pytest-tricks#31]
I can see this question coming up a lot :wink:

Possible solutions include:

- Returning a helper object/function from the fixture
- Adding a marker on the test and reading that from the fixture

**Comment by hackebrot** [comment]
Sounds like a great article for pytest-tricks! 👍 

Do you have time to write a something?

**Comment by The-Compiler** [comment]
Nope, sorry - too busy with university and other OSS stuff, and from what it looks like this isn't going to change anytime soon...

**Comment by hackebrot** [comment]
😢
